tests/test_dia_calculation_plugins.py

```diff
@@ -443,39 +603,51 @@ def testCalculate(self):
         """Test flux maximum slope.
         """
         n_sources = 10
-        diaObject = dict()
+        objId = 0
+
+        # Test max slope value.
         fluxes = np.linspace(-1, 1, n_sources)
         times = np.linspace(0, 1, n_sources)
```

Comment:
append a value so the max slope is different than the slope of all the other points?